akka-sample-distributed-data-scala/src/main/scala/sample/distributeddata/ShoppingORCart.scala

```diff
@@ -0,0 +1,143 @@
+package sample.distributeddata
+
+import java.util.UUID
+
+import scala.concurrent.duration._
+import akka.actor.Actor
+import akka.actor.ActorRef
+import akka.actor.Props
+import akka.cluster.Cluster
+import akka.cluster.ddata.{DistributedData, ORMultiMap, ORMultiMapKey}
+
+import scala.collection.immutable.HashSet
+
+object ShoppingORCart {
```

Comment:
Would be great with a short scaladoc summary up here (I know the other ones doesn't have one), both for maintainability and how easy it is to understand it.